assignment/scripts/function-practice.js

```diff
@@ -10,74 +10,148 @@ function hello() {
 }
 // Call the function to test
 console.log('Test - should say "Hello World!"', hello());
+console.log(hello());
 
-
-// 2. Function to return an personalized hello, using the `name` argument.
+// 2. Function to return a personalized hello, using the `name` argument.
 //    for example 'Hello, Jo!', or 'Hello, Stacy!'
-function helloName( name ) {
-  return;
+function helloName(name) {
+  console.log(`Hello, ${name}!` );
 }
 // Remember to call the function to test
-
+helloName('Jose');
 
 // 3. Function to add two numbers together & return the result
-function addNumbers( firstNumber ) {
+function addNumbers(firstNumber, secondNumber) {
+  // return firstNumber + secondNumber;
+  return firstNumber + secondNumber;
+}
+let sum = addNumbers(5, 17);
+console.log(sum);
+// or
+console.log(addNumbers(9, 2));
+
+// or 
+
+let _addNumbers = (firstNumber, secondNumber) => {
   // return firstNumber + secondNumber;
+  return firstNumber + secondNumber;
 }
+console.log(_addNumbers(6, 10));
 
+// or
+
+let __addNumbers = (firstNumber, secondNumber) => firstNumber + secondNumber; 
+console.log(__addNumbers(6, 20));
 
 // 4. Function to multiply three numbers & return the result
-function multiplyThree( ){
+function multiplyThree(x, y, z) {
+  return x * y * z;
+}
+let product = multiplyThree(5, 5, 5);
+console.log(product);
+// or 
+console.log(multiplyThree(2, 2, 2));
+
+// or 
 
+let _multiplyThree = (x, y, z) => {
+  return x * y * z;
 }
+console.log(_multiplyThree(3, 3, 3));
 
+// or
+
+let __multiplyThree = (x, y, z) => x * y * z;
+console.log(__multiplyThree(4, 4, 4));
+
+let squared = x => x^2;
+console.log(squared(2));
 
 // 5. Function that will return true if a number is positive, 
 //    or greater than zero, and false otherwise
-function isPositive( number ) {
+function isPositive(number) {
   if ( number > 0 ){
-    return;
+    return true;
   }
-    return;
+    return false;
 }
+
 // Call the function to test each outcome (true & false) 
 // Write a separate console.log statement for each outcome
 console.log( 'isPositive - should say true', isPositive(3) );
 console.log( 'isPositive - should say false', isPositive(0) );
 console.log( 'isPositive - should say false', isPositive(-3) );
 
-
 // 6. Function to return the _last_ item in an array. If the 
 //    array is empty, return `undefined`.
-function getLast( array ) {
+function getLast(array) {
+  if (array.length === 0) {
+    return undefined;
+  } else {
+    return array[array.length - 1];
+  }
+}
+console.log(getLast([2,4,99]));
 
+// or
+
+let _getLast = array => {
+  if (array.length === 0) {
+    return undefined;
+  } else {
+    return array[array.length - 1];
+  }
 }
+console.log(_getLast([2,4,55]));
+
+// or
+
+let __getLast = array => array[array.length -1]; 
```

Comment:
I love how much you are practicing the different function syntaxes. This is a great thing to get comfortable with.